src/main/java/org/broadinstitute/hellbender/tools/coveragemodel/CoverageModelWLinearOperatorLocal.java

```diff
@@ -0,0 +1,102 @@
+package org.broadinstitute.hellbender.tools.coveragemodel;
```

Comment:
this is regularizer-related and will be removed/significantly updated/refactored soon.